LocalSettings.php

```diff
@@ -710,6 +710,8 @@
 			'mem\d{1,2}',
 			'miraheze\d{0,2}',
 			'sslrequest',
+			'jobchron\d{0,2}',
+			'task\d{0,2}',
```

Comment:
I think that this should be `mwtask\d{0,2}`.

Reply:
Fixed when resolving merge conflict